Add Greedo's letter to the site

donut211 opened the pull request.

donut211 committed:
Add files via upload

donut211 committed:
Merge pull request #1 from donut211/Add-Greedo's-Letter-to-website!!!!!

Add files via upload

errinlarsen commented on collections/_posts/LetterToTheDuke:
```diff
@@ -0,0 +1,18 @@
+To the Duke of the Duchy of Erik,
```
missing front matter...

```suggestion
---
author: Greedo
title: Letter to the Duke
date: 2020-12-08 16:29:23 -0700
categories: pc greedo journals 
---
To the Duke of the Duchy of Erik,
```

errinlarsen reviewed (changes requested):
For future reference, you missed a few things in this pull request:

1. "Posts" need to be named like, `2020-12-08-letter-to-the-duke.md` (i.e. with the date in that specific format at the front of the file name, and using dashes)
2. Jekyll uses "front matter" in its Markdown files. The front matter is contained within `---` lines in the file, and the front matter itself is in YAML format (see the example, below). It's called "front matter" 'cause those line contained within `---` must go at the very top of your file.
  ```YAML
   ---
   author: Greedo
   title: Letter to the Duke
   date: 2020-12-08 16:29:23 -0700
   categories: pc greedo journals 
   ---
   ```

errinlarsen committed:
Update Letter to the Duke for Jekyll

1. "Posts" need to be named like, `2020-12-08-letter-to-the-duke.md`
  - (i.e. with the date in that specific format, at the front of the
    file name, and using dashes)
2. Jekyll uses "front matter" in its Markdown files. The front matter is
  contained within `---` lines in the file, and the front matter itself
  is in YAML format (see the example, below). It's called "front matter"
  'cause those line contained within `---` must go at the very top of
  your file.

```YAML
 ---
 author: Greedo
 title: Letter to the Duke
 date: 2020-12-08 16:29:23 -0700
 categories: pc greedo journals
 ---
 ```
